pkg/gardenlet/controller/extensions/extensions.go

```diff
@@ -83,23 +92,22 @@ func (c *Controller) Initialize(ctx context.Context, seedClient kubernetes.Inter
 // Initialize must be called before running the controller.
 func (c *Controller) Run(ctx context.Context, controllerInstallationWorkers, shootStateWorkers int) {
 	if !c.initialized {
-		c.log.Fatal("controller is not initialized")
+		c.log.Info("Controller is not initialized, cannot run it")
```

Comment:
You're right, in this case we want it to panic since it's a programmer-level error not to call `Initialize` first. I now explicitly `panic` instead of logging this.